db/migrate/20171122092336_add_ignored_fact_names_setting.rb

```diff
@@ -0,0 +1,14 @@
+class AddIgnoredFactNamesSetting < ActiveRecord::Migration
+  def up
+    Setting.create!(
+      Setting.set(
+        'ignored_fact_names',
+        N_("Ignore fact names that match these values during facts importing, you can use * wildcard to match names with indexes e.g. macvtap*"),
+        Setting[:ignored_interface_identifiers],
+        N_('Ignore fact names with matching identifier'))) if Setting[:ignored_interface_identifiers]
+  end
+
+  def down
+    Setting.where(:name => :ignored_fact_names).destroy_all
+  end
+end
```

Comment:
One thing - this is a great opportunity to clean those facts from db. There are two choices:

A) Implicit - simply delete them in the migration. Can be dangerous for existing users.

B) Explicit - create a rake task and we can ask customer to run it (first show the facts, then confirm deletion). This will be safe enough for everybody.

It would be very nice to have the same rake task for NICs.